pkg/webhook/admission/http.go

```diff
@@ -69,7 +69,7 @@ func (wh *Webhook) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// verify the content type is accurate
 	if contentType := r.Header.Get("Content-Type"); contentType != "application/json" {
 		err = fmt.Errorf("contentType=%s, expected application/json", contentType)
-		wh.getLogger(nil).Error(err, "unable to process a request with an unknown content type", "content type", contentType)
+		wh.getLogger(nil).Error(err, fmt.Sprintf("unable to process a request with unknown content type: %q", contentType))
```

Comment:
I don't think we need a k/v pair for content type. Doesn't seem to make a lot of sense for log filtering